sample_project/Assets/SampleViewer/Samples/FeatureLayer/readme.md

```diff
@@ -48,9 +48,7 @@ After the request succeeds we will parse the text and turn it into meaningful `G
 Feature layers sometimes only hold the feature's X/Y position but not the Altitude. Because of this, this sample has some added complexity to show you how to place the object on the ground. There are two recommended ways to get the elevation at a given point.
 
 1. Use the [elevation's REST API](https://developers.arcgis.com/rest/services-reference/enterprise/image-service.htm) to query or identify a point and get the elevation value.
-2. Use the game engine's [raycast](https://docs.unity3d.com/ScriptReference/Physics.Raycast.html) feature from a point above the surface towards the ground to detect the altitude dynamically.
-
-This sample uses option 2. A caveat of using option 2 is that the raycast will only hit the surface if the point we are trying to hit is being loaded. The ArcGIS Maps SDK for game engine's dynamically stream content based on what the camera is seeing. To account for this `FeatureData.cs` only casts a ray for a stadium when it is within the `RayCastDistanceThreshold` which defaults to 30 km. To keep the CPU from getting overloaded, we stop adjusting the object's altitude after the raycast hits the ground. The way we use raycast in this sample is just one way of accomplishing this task. Feel free to experiment to find a way that works best for your use case.
+2. Set the Surface Placement mode on the ArcGISLocation Component to `OnTheGround`.
```

Comment:
I think this entire section can be removed since with the new surface placement using an external API wouldn't make sense